app/views/record/defer/did-trainee-start.html

```diff
@@ -0,0 +1,21 @@
+{% extends "_templates/_record-form.html" %}
+
+{% set pageHeading %}
+  Did the trainee start their {{ "ITT" if record | courseDatesAreAmbiguous else "course" }}?
```

Comment:
```suggestion
  Did the trainee start their {{ "ITT" if data.record | courseDatesAreAmbiguous else "course" }}?
```

In the context of this template, `record` doesn't yet exist - so I think this will always render as `course`